src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
I will recond here composition instead of inheritance, because relation when u a using inheritance should be "is", but here Lottery is not ColorSupplier, here is better relation has-a(composition)